packages/drivers/odsp-driver/src/fetchSnapshot.ts

```diff
@@ -430,6 +430,9 @@ function getFormBodyAndHeaders(
     if (odspResolvedUrl.shareLinkInfo?.sharingLinkToRedeem) {
         formParams.push(`sl: ${odspResolvedUrl.shareLinkInfo?.sharingLinkToRedeem}`);
     }
+    if (odspResolvedUrl.isClpCompliantApp) {
```

Comment:
@jatgarg Is this what you meant by needing to be part of the form body to avoid the options call?

Reply:
So this change is not required. All the calls go through epoch tracker where you can add the header in form body or as a query param or normal header. So we can remove this change. If you see in epoch tracker we have conditions on how to add this.

Reply:
Removed this change.